phpcs.xml.dist

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<ruleset name="WP-CLI-maintenance-mode">
+	<description>Custom ruleset for WP-CLI maintenance-mode-command</description>
+
+	<!--
+	#############################################################################
+	COMMAND LINE ARGUMENTS
+	For help understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	For help using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	#############################################################################
+	-->
+
+	<!-- What to scan -->
+	<file>.</file>
+
+	<!-- Ignoring select files/folders
+		 https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders -->
+	<exclude-pattern>*/tests/data/*</exclude-pattern>
```

Comment:
Please remove this line - the `tests/data` directory does not exist in this repo and should not be excluded by default accidentally if it would be added in the future.

Reply:
Ah, good catch.

Reply:
Sure @jrfnl, will do.

Reply:
@jrfnl and @schlessera I have removed `tests/data` directory and rebased.